src/ucp/core/ucp_rkey.c

```diff
@@ -36,10 +36,9 @@ ucs_status_t ucp_rkey_pack_uct(ucp_context_h context,
 
     /* Write both size and rkey_buffer for each UCT rkey */
     uct_memh_index = 0;
-    for (md_index = 0; md_index < context->num_mds; ++md_index) {
-        if (!(md_map & UCS_BIT(md_index))) {
-            continue;
-        }
+    ucs_for_each_bit (md_index, md_map) {
+        ucs_assert(md_index < context->num_mds);
```

Comment:
don't think we need these assertions here.. since we have a test for ucs_for_each_bit